robots/iiwa.urdf.xacro

```diff
@@ -3,6 +3,7 @@
   <xacro:arg name="robot" default="iiwa7"/>
   <xacro:arg name="prefix" default="$(arg robot)_"/>
   <xacro:arg name="robot_ip" default="192.170.10.1"/>
+  <xacro:arg name="fri_port" default="30200"/>
```

Comment:
I wonder if it would make sense to just name that `port` or something similar. I can see other robots that might need both `robot_ip` and `port` number without it being related to FRI. But not sure if we need to standardize here actually.

Reply:
`port`is fine too